usr/sbin/orjail

```diff
@@ -432,135 +400,136 @@ if ! ip netns list | eno grep -e \\b"$NAME"\\b; then
 
   print G " * Creating a $NAME namespace..."
   # add network namespace
-  ip netns add "$NAME" || die "Failed to add a new namespace"
+  ip netns add "$NAME"
```

Comment:
set -e is OK but here we don't have the correct error message anymore...

Reply:
solved here e37c7f9118d55b11086a6183b1e26b7972fe6aff